plutus-pab/plutus-pab.cabal

```diff
@@ -157,6 +157,7 @@ library
         network -any,
         network-mux -any,
         newtype-generics -any,
+        nothunks -any,
```

Comment:
I was thinking that this is probably the successor of all the `NoUnexpectedThunks` classes.

Reply:
Yep, and we'll be using it elsewhere before long.